test/logged_smtp_delivery_test.rb

```diff
@@ -31,7 +31,15 @@ def welcome(extra={})
     }
     let(:smtp) { Mailcrate.new(port) }
     let(:port) { 25552 }
-    let(:mail) { smtp.mails.last }
+
+    def mail
```

Comment:
I believe this is required because `@mails` in Mailcrate is not populated until after the connection is closed, so there's a chance the test can execute before `mail` is available. I plan to file an issue in the mailcrate repo for this.